### ISSUE TITLE
Create two bars across the cabin for the pontoon and amphibian variants

### ISSUE THREAD

**Issue by gilbertohasnofb** [Juanvvc/c172p-detailed#313]
Create two bars across the cabin for the pontoon and amphibian variants, such as seen in these images:

http://islandaero.com/images/70-172dbp.jpg

http://www.parkrapidsaviation.com/userfiles/image/IMG_4538%281%29.JPG

Possibly related to #206

**Comment by tigert** [comment]
Not the perfect view but visible from different angle still: http://tigert.1g.fi/kuvat/MIK/2006-07-30-parnu/_DSC2644.jpg

**Comment by tigert** [comment]
Looks like there is a flat metal bracket that the bolts attach to and which is bolted to the glare area of the fuselage?

 |__\\//__|

**Comment by gilbertohasnofb** [comment]
@tigert Very nice, thanks for the additional information!

According to your picture, it looks like we will need to raise our compass as well as the two bars connect below it. To be honest, I always thought our compass should have some sort of mount below it.

Also, I suggest we only implement this after solving #6 and #118 as they may influence the position of the panel.

Finally, once this work is done I can create some nice textures for these bars. For instance, in the case of OH-CTL in the image tigert posted above, the bars have their painting quite scratched, but some liveries (such as the shiny 4X-CHV) could have them in better conservation.

**Comment by onox** [comment]
http://tigert.1g.fi/kuvat/MIK/2006-07-30-parnu/_DSC2651.jpg :open_mouth: 

~~We~~ @thevirtualfer can add still much more detail to the model :smile:

**Comment by gilbertohasnofb** [comment]
@onox I don't understand the connection between the topic and the picture you posted

**Comment by onox** [comment]
There's no connection :stuck_out_tongue_winking_eye:

**Comment by gilbertohasnofb** [comment]
Oh :smile:

**Comment by onox** [comment]
Here's another picture of the two bars: http://tigert.1g.fi/kuvat/MIK/2006-07-30-parnu/_DSC2673.jpg

**Comment by gilbertohasnofb** [comment]
Very nice! Now we can see better how the compass look like and how it's mounted in the cockpit!

**Comment by wkitty42** [comment]
FWIW: i've seen these support braces in some craft's pics and not in others... i haven't paid attention if they are only floatplanes where i've seen them, though...

@gilbertohasnofb on the compass position, take a close look at ours and theirs... ours is mounted to the front of the glare panel whereas theirs is mounted back closer to the windshield... with the braces in place, and their compass being so far back, it only makes sense for them to have raised it up and placed it between or mounted onto the braces... looking at it with my eyes, it looks to me like our compass is far enough forward that the braces would be completely behind it and it should be able to remain where it is... i thought i'd toss that out there but it isn't my decision to make...

**Comment by gilbertohasnofb** [comment]
So are there any 172 with compasses like ours, directly on the panel and without a mount? I thought that our compass was actually badly designed, but if that's a realistic option then by no means let's no touch it! I am just not sure about it.

**Comment by wkitty42** [comment]
TBH: i don't know... will have to try to find more interior pics... you guys have posted numerous ones during the panel reconstructions but i don't know where they are or how you found them...

**Comment by gilbertohasnofb** [comment]
Most of them came from tigert's websit: http://tigert.1g.fi/kuvat/Simulators/Flightgear/C172-dimensions/

Others were found by googling or at airliners.net. I have tons of cockpit pictures saved on my computer (from those sources mentioned) and absolutely all of them share a same model of compass which in turn is different then ours. Ours is a simple rectangle while theirs is rounded, has a mount, etc. See the image by tigert above.

Therefore I think it would be better to rework our compass. I will open an issue for that.

**Comment by gilbertohasnofb** [comment]
Forget about what I just wrote, I found a single counterexample to what I posted. Here is a compass which is identical to ours: https://s-media-cache-ak0.pinimg.com/736x/fb/4b/40/fb4b409d723a1529a1dbd947851077dd.jpg

So even though they seem to be much rarer than the one in tigert's images, they do exist and I think we shouldn't necessarily change it.

**Comment by wkitty42** [comment]
while we're talking about the compass, can anyone tell me (or any of us) why it is ""clickable""?? i mean, it shows up with the CTRL-C hilites but i don't know why it should or how i should be able to adjust it... isn't it just a simple magnetic compass made of a dome in a oil bath housing similar to a flat hand-held compass you might by to go camping and hiking with? i've never seen or known of any way to adjust them so i don't understand why this one (from the original craft and carried over in this one) shows up with CTRL-C...

**Comment by gilbertohasnofb** [comment]
I don't know, it doesn't make sense to me either as the compass is not clickable. Now why are all the screws clickable and they open the checklist, please? That makes no sense to me at all...

**Comment by wkitty42** [comment]
i saw all those screws come in doing that, too... it was already done by the time i got the fork and other stuff done... i was following developments from the beginning but it was several weeks before i forked and started participating more actively...

**Comment by onox** [comment]
@gilbertohasnofb Yeah, almost all screws are clickable. I think just one screw is sufficient (so you can open the checklists without having to look down). Clickable compass makes no sense either.

**Comment by onox** [comment]
Can we make a new issue for this?

**Comment by gilbertohasnofb** [comment]
> I think just one screw is sufficient (so you can open the checklists without having to look down)

Really? That is really cryptic and will look like a bug. Come on, we can open the checklist via the menu, via a keyboard shortcut and also by looking down and clicking on it, do we really need still another way? Also, using checklist is about NOT being lazy, so looking down to grab it is hardly a problem, is it? :smile: 

> Can we make a new issue for this?

I will do that. Just let's first decide if we are leaving one clickable screw or no clickable screws at all.

**Comment by wkitty42** [comment]
personally speaking, i don't know when they appeared or why whoever did them did them... i was surprised the first time i hit CTRL-C and then saw everything light up with all the screws being clickable... then i saw the tool tip... some time later i finally saw the side pocket on the door which is/was a nice addition... i've always used the menu but i've only been doing FGFS for 5 or 6 months... i don't have any idea what the keyboard shortcut may happen to be...

so anyway, to the question: "are leaving one clickable screw or no clickable screws at all"? i'd say no clickable screws... they just don't seem "life like"...

**Comment by onox** [comment]
> Also, using checklist is about NOT being lazy, so looking down to grab it is hardly a problem, is it?

Haha, true. No screws then.

**Comment by gilbertohasnofb** [comment]
> Haha, true. No screws then.

Great! So I am opening a new issue for this now.

**Comment by gilbertohasnofb** [comment]
See #315 and #316

**Comment by tigert** [comment]
You are right, the wet compass is not adjustable. It could be useful to have a larger-size popup of the compass when you click it since the dial is so small, though. Maybe it used to have something like that and the clickspot is still there?

**Comment by wlbragg** [comment]
Linked to issue #206

**Comment by wlbragg** [comment]
![fix2](https://cloud.githubusercontent.com/assets/11069680/10960801/ebb3cb2e-834c-11e5-8e2d-b689eb68f6da.jpg)

**Comment by tigert** [comment]
:+1: !

**Comment by gilbertohasnofb** [comment]
Looks really good! :smile:

**Comment by onox** [comment]
Looks nice. Maybe add a little piece of metal between the two endpoints (to connect them) like in https://github.com/Juanvvc/c172p-detailed/issues/313#issuecomment-113240399

**Comment by wlbragg** [comment]
>Maybe add a little piece of metal between the two endpoints 

Will do, I didn't even notice that little piece.
